static/js/biomedical/protein/page.tsx

```diff
@@ -83,39 +94,50 @@ export interface InteractingProteinType {
 export class Page extends React.Component<PagePropType, PageStateType> {
   constructor(props: PagePropType) {
     super(props);
-    this.state = { data: null };
+    this.state = {
+      data: null,
```

Comment:
nit: you might not need to keep data in the state anymore since you're keeping all the processed data in the state?

Reply:
we're using `this.state.data` for protein description and protein link in lines 148-149!